### PR TITLE
Bug

### LINKED ISSUE #33: 'az exit'/'az quit' doesn't exit the shell or does anything meanful
both show up in the combo when i type `az` and the description shows 'exit the program'. I figure it means to exit the shell? but end up getting the output of the 'az -h'.

### PR DESCRIPTION
**Pull request by oakeyc**
Fixes #33

### CODE REVIEW

File: azclishell/app.py [hunk 1]
```diff
@@ -393,7 +393,7 @@ def _special_cases(self, text, cmd, outside):
         if self.default_command:
             cmd = self.default_command + " " + cmd
 
-        if text.strip() == "quit" or text.strip() == "exit":
+        if cmd.strip() == "quit" or cmd.strip() == "exit":
```
Comment (tjprescott) [inline azclishell/app.py:396]: What is the difference between `text` and `cmd`?

Comment (oakeyc) [reply azclishell/app.py:396]: I use cmd to mean more of what the user wanted as a command versus the text is what's on the line. In this specific example, I strip 'az' from the cmd but not the text, so what I to know is if they wrote 'az quit' versus if the line said 'quit.'

Comment (tjprescott) [reply azclishell/app.py:396]: So it seems to me that as I start typing... text and cmd would be the same... and then once the shells basically switches over to parameter completions, then cmd is fixed even though text grows... is it something like that or not even?

Comment (oakeyc) [reply azclishell/app.py:396]: A little. The text variable should always be what's represented on the line, but the cmd is reformatted for what the shell needs for information

**Review by derekbekoe — APPROVED** [review]

**Pull request merged**